### PR TITLE
MJ-168 Landing page fixes

### PR DESCRIPTION
**Pull request by AniaPazniewska**

### CODE REVIEW

File: client/src/components/landing/LandingNavigationBar.vue [hunk 1]
```diff
@@ -1,8 +1,10 @@
...
   <div class="mj-landing-navbar">
-    <application-title
-      :is-nav-bar="true"
-      :with-icon="true" />
+    <a onclick="window.location.reload()">
+      <application-title
+        :in-nav-bar="true"
+        :with-icon="true" />
+    </a>
     <div class="mj-landing-navbar-buttons">
```
Comment (juliapacula) [inline client/src/components/landing/LandingNavigationBar.vue]: I would just remove this code. Just add "cursor: pointer". `a` with event `onclick` is worse. `a` should handle `href` param to be treated as a link. Or use vue's `@click`, just don't use plain js when we use a framework.

File: client/src/i18n/locales/en.json [hunk 2]
```diff
@@ -38,6 +38,8 @@
     "AboutMistakesJournal": {
       "HeaderFirstPart": "Get the most out of your mistakes",
       "HeaderSecondPart": "by keeping a",
+      "EarlyAccessHeader": "Welcome to Early Access!",
+      "EarlyAccessText": "See for yourself and tell us what you think!",
```
Comment (juliapacula) [inline client/src/i18n/locales/en.json]: Too many `!` for me. I would remove `!` from `EarlyAccessText` and just ad regular `.`

File: client/src/i18n/locales/pl.json [hunk 3]
```diff
@@ -38,15 +38,17 @@
     "AboutMistakesJournal": {
       "HeaderFirstPart": "Wykorzystaj w pełni swoje błędy",
       "HeaderSecondPart": "prowadząc",
+      "EarlyAccessHeader": "Witamy we wczesnym dostępie!",
+      "EarlyAccessText": "Przekonaj się sam i daj znać do myślisz!",
       "Step1": {
         "StepName": "Rozpoznaj",
         "Header": "Błędy są cenne",
-        "Text": "Pomożemy Ci zapisywać oraz organizować Twoje błędy, tak aby czerpać z nich jak najwięcej."
+        "Text": "Pomożemy Ci zapisywać oraz organizować Twoje błędy, tak abyś mógł z nich czerpać jak najwięcej."
```
Comment (juliapacula) [inline client/src/i18n/locales/pl.json]: I specially changed the text. Not every user is male :)

File: client/src/i18n/locales/pl.json [hunk 4]
```diff
@@ -38,15 +38,17 @@
...
       },
       "Step2": {
         "StepName": "Analizuj",
         "Header": "Znajdź rozwiązania",
-        "Text": "Niektóre błędy nie są proste, ale zrobimy co w naszej mocy, aby pomóc Ci zreflektować się nad nimi, tak abyś miał porady na przyszłość."
+        "Text": "Niektóre błędy nie są proste ale zrobimy co w naszej mocy, aby pomóc Ci zreflektować się nad nimi, tak abyś miał porady na przyszłość."
```
Comment (juliapacula) [inline client/src/i18n/locales/pl.json]: Why removing `,`? I suppose in polish you use `,` before `ale`.

File: client/src/components/landing/LandingNavigationBar.vue [hunk 5]
```diff
@@ -51,9 +53,13 @@ export default Vue.extend({
   &-buttons {
     display: flex;
 
-    > :not(:last-child) {
+    > * {
```
Comment (juliapacula) [inline client/src/components/landing/LandingNavigationBar.vue:55]: Why last child should have margin right?

File: client/src/components/landing/AboutMistakesJournal.vue [hunk 6]
```diff
@@ -76,9 +96,15 @@
           <span class="step-text">{{ $t('LandingPage.AboutMistakesJournal.Step3.Text2') }}</span>
         </div>
         <img
+          v-if="$i18n.locale === 'EN'"
+          alt="Mistakes journal front page"
+          class="mj-landing-steps-image"
+          src="@/assets/icons/mistakes_en.svg">
```
Comment (juliapacula) [inline client/src/components/landing/AboutMistakesJournal.vue]: With all those images I would do something like

```
        <img
          v-if="$i18n.locale === 'EN'"
          alt="Mistakes journal front page"
          class="mj-landing-steps-image"
          :src="require(@/assets/icons/' + $i18n.locale + '/mistakes.svg)">
```
and place the images in respective dirctories: `icons/PL` and `icons/EN`, without language name in the file.

**Review by juliapacula — CHANGES_REQUESTED** [review]

File: client/src/i18n/locales/pl.json (hunk 3 above)
Comment (AniaPazniewska) [reply client/src/i18n/locales/pl.json]: Szymons request so i dont know what to do now

File: client/src/i18n/locales/pl.json (hunk 4 above)
Comment (AniaPazniewska) [reply client/src/i18n/locales/pl.json]: Szymons translation

File: client/src/i18n/locales/en.json (hunk 2 above)
Comment (AniaPazniewska) [reply client/src/i18n/locales/en.json]: Szymons translation

File: client/src/components/landing/LandingNavigationBar.vue (hunk 5 above)
Comment (AniaPazniewska) [reply client/src/components/landing/LandingNavigationBar.vue:55]: Szymons request to be margin in the end

**Review by juliapacula — APPROVED** [review]

**Pull request merged**